person.rb

```diff
@@ -20,6 +23,15 @@ def correct_name
     @name
   end
 
+  attr_accessor :classroom
+
+  # many to many relationship
+  attr_reader :rentals
+
+  def add_rental(date, book)
+    Rental.new(date, self, book)
```

Comment:
- [ Required ] I think it would be better if you pass arguments to the `Rental` class constructor as they are listed in the definition of class `Rental`. In that case you should pass `date`, `book` and lastly `person`, please note you interchanged `book` and `person`

## class Rental
```ruby
class Rental
  attr_accessor :date, :book, :person

  def initialize(date, book, person)
    @date = date
    @book = book
    @person = person
    book.rentals << self
    person.rentals << self
  end
end
```

## Method `add_rental` in person.rb
```ruby
  def add_rental(date, book)
    Rental.new(date, book, self)
  end
```